ports/mimxrt/boards/MAKERDIARY_RT1011_NANO_KIT/mpconfigboard.h

```diff
@@ -0,0 +1,87 @@
+#define MICROPY_HW_BOARD_NAME "iMX_RT1011_Nano_Kit"
+#define MICROPY_HW_MCU_NAME   "MIMXRT1011DAE5A"
+#define MICROPY_HW_USB_MANUFACTURER_STRING "Shenzhen Zaowubang Technology Co.,Ltd."
```

Comment:
Maybe this can be "Makerdiary" instead?  I don't really mind either way.

Reply:
Changed. Shorter is better most of the time.